scout/server/blueprints/institutes/templates/overview/clinvar_submissions.html

```diff
@@ -191,9 +193,12 @@ <h4>Case data:</h4>
             </thead>
             <tbody>
               {% for case in subm_obj.case_data %}
-                <tr>
+                <tr class="text-center">
                   <td width="5%">{{loop.index}}</td>
-                  <td width="15%">{{case.individual_id}}</td>
+                  <td width="20%">
+                    <input type="text" id="{{loop.index}}" value="{{case.individual_id}}">
```

Comment:
😄 Nice jinja use - I learn. But you can only safely do that once per page without adding some unique string to the id..

Reply:
That would happen if you have more then one submission, right. Let me see if I can find a solution!

Reply:
Fixed by having a different form in the template, that is passing exactly the stuff I want, no more loop number